app/components/ui/links.tsx

```diff
@@ -21,3 +21,16 @@ export function Link(props: LinkProps) {
     </div>
   );
 }
+
+type DefaultLinkProps = RemixLinkProps & {
+  children: ReactNode;
+  to: string;
+};
+
+export function DefaultLink({ to, children, ...props }: DefaultLinkProps) {
```

Comment:
Would it be possible to merge `DefaultLink` and `Link` into the same component to reduce confusion? I feel it makes more sense that `Link` is the default link component we use in our app.

Reply:
Agree, the name looks confusing 😅 
Will look at it!